articles/communication-services/concepts/call-automation/call-automation.md

```diff
@@ -42,7 +42,7 @@ The following list presents the set of features that are currently available in
 |                       | Place new outbound call to one or more endpoints  | ✔️    | ✔️    |     ✔️         |    ✔️   |
 |                       | Redirect* (forward) a call to one or more endpoints  | ✔️    | ✔️    |     ✔️         |    ✔️   |
 |                       | Reject an incoming call                           | ✔️    | ✔️    |     ✔️         |    ✔️   |
-| Mid-call scenarios    | Add one or more endpoints to an existing call     | ✔️    | ✔️    |     ✔️         |    ✔️   |
+| Mid-call scenarios    | Add one or more endpoints to an existing call (with custom context) | ✔️    | ✔️    |     ✔️         |    ✔️   |
```

Comment:
We will likely support more actions in the future, so how about creating new line for custom context instead of updating the Add or transfer line item? Something like this: 

Share [custom info](https://learn.microsoft.com/en-us/azure/communication-services/how-tos/call-automation/custom-context) (via SIP headers) with endpoints when adding them to a call or transferring a call to them.

Reply:
updated! ty!